src/js/components/Peak/PeakStatistics.tsx

```diff
@@ -1,80 +1,60 @@
-} from '../../API/peaks/methods';
-import { User } from '../../models/interfaces';
-import { Icon } from '../common';
+/* eslint-disable @typescript-eslint/no-explicit-any */
+import React from 'react';
 import styles from './PeakStatistics.module.css';
+import UserInfo from './UserInfo';
 
-const PeakStatistics = ({ peakId }: { peakId: string }) => {
-  const [firstConqueror, setFirstConqueror] = useState<User | undefined>(
-    undefined
-  );
-  const [lastConqueror, setLastConqueror] = useState<User | undefined>(
-    undefined
-  );
-  const [numberOfPeakConquerors, setNumberOfPeakConquerors] = useState<
-    number | undefined
-  >(undefined);
-  const [averageTimeCompletion, setAverageTimeCompletion] = useState<
-    number | undefined
-  >(undefined);
-
-  useEffect(() => {
-    const getPeakStatistics = async () => {
-      setFirstConqueror(await getFirstConqueror(peakId));
-      setNumberOfPeakConquerors(await getNumberOfPeakConquerors(peakId));
-      setAverageTimeCompletion(await getPeakAverageTimeCompletion(peakId));
-      setLastConqueror(await getLastConqueror(peakId));
-    };
-    getPeakStatistics();
-  }, []);
-
+const PeakStatistics = ({ statistics }: { statistics: any }) => {
   return (
     <>
-      <h3 className={styles.statisticsHeader}>Statistics</h3>
-      <p className={styles.wrapper}>
-        Pierwszy zdobywca:
-        {firstConqueror ? (
-          <>
-            <Icon
-              url={firstConqueror.profilePhoto}
-              variant="xs"
-              className={styles.horizontalMargin}
-            />
-            <span>
-              {firstConqueror.firstName} {firstConqueror.lastName}
-            </span>
-          </>
-        ) : (
-          <span className={styles.horizontalMargin}>-</span>
-        )}
-      </p>
-      <p className={styles.wrapper}>
-        Ostatni zdobywca:
-        {lastConqueror ? (
-          <>
-            <Icon
-              url={lastConqueror.profilePhoto}
-              variant="xs"
-              className={styles.horizontalMargin}
-            />
-            <span>
-              {lastConqueror.firstName} {lastConqueror.lastName}
-            </span>
-          </>
-        ) : (
-          <span className={styles.horizontalMargin}>-</span>
-        )}
-      </p>
-      {numberOfPeakConquerors !== undefined && (
-        <p>Liczba zdobywców: {numberOfPeakConquerors}</p>
+      <h3 className={styles.statisticsHeader}>Statystyki</h3>
+      {statistics.time_average !== undefined && (
+        <p>Średni czas zdobycia szczytu: {statistics.time_average} min</p>
+      )}
+      {statistics.completion_total !== undefined && (
+        <p>Liczba zdobywców: {statistics.completion_total}</p>
+      )}
+      {statistics.completion_first && (
+        <>
+          Pierwszy zdobywca:
+          <UserInfo
+            profilePhoto={statistics.completion_first.user.profilePhoto}
+            name={
+              statistics.completion_first.user.firstName +
```

Comment:
Backend nie powinien tego odsyłać w camelCase?

Reply:
Może i powinien, ale odsyła tak

Reply:
Pewnie coś jest popsute w snake_case na backu, bo profilePhoto jest we właściwym case. Z jednej strony nie ma większego znaczenia, z drugiej ktoś może się przejechać

Reply:
![image](https://user-images.githubusercontent.com/56346754/120239635-fb136a00-c25e-11eb-82ab-6531767d82ba.png)

Do mapy po prostu dali klucze używając snake_case, a cała reszta się ładuje z bazy, więc jest cammelCase.

Reply:
Jak chcesz to podmień te klucze na camelCase, jak nie to zostaw